src/badge/badge.js

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
 
 let styles = {};
 
 const Badge = props => {
   const { badge } = props;
 
+  const {
+    badgeContainerStyle,
+    badgeTextStyle,
+    onPress,
+    element,
+    value
+  } = props;
+
   if (!badge) throw Error('badge prop is required');
 
-  if (badge.element) return badge.element;
+  if (element) return badge.element;
```

Comment:
This is weird, basically we are letting user pass a whole element and returning that. If they really need to do that and not utilize badge styles then why use badge in the first place. I don't think this is needed @monte9 thoughts?